vim/merlin/autoload/merlin.py

```diff
@@ -88,6 +88,9 @@ def concat_map(f, args):
 
 def current_context():
     filename = vim.eval("expand('%:p')")
+    if platform == "cygwin":
```

Comment:
Are you sure this is the right check? The code could be running on Cygwin and being used with a native Windows port of `merlin` (your case) or using the Cygwin-compiled version of `merlin` (which would need to use usual Unix paths). At the moment I am not sure how best to differentiate between these two cases... One possibility could be to use `ldd` and check if the merlin executable depends on `cygwin1.dll`, but that rather ugly...